ApiTests/Tests/API/APITests.cs

```diff
@@ -5,30 +5,33 @@ namespace TestProject.Tests.API
 {
     public class Tests
     {
+        private Pet pet;
+
         [SetUp]
         public void Setup()
         {
             //move test data creation here
+            pet = new Pet(420000, "Jake", "Alive");
```

Comment:
that'd be better to use config file, the one that is already in the project
there is a ConfigReader class that has some methods that might help with that